### ISSUE TITLE
Reproduced code results

### ISSUE THREAD

**Issue by gkjgkd** [peterant330/KUEA#6]
![Image](https://github.com/user-attachments/assets/ad608f5d-2101-4cbb-ac58-d91e876639d4)
when I reproduced code result, the training loss is diffrent from the paper, can you provide your training log?
thank you
